File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/create-game/create-game-response-dto.ts
import type { BaseDto } from "~/interfaces/api/contracts/base/dto/base-dto";

export interface CreateGameResponseDto extends BaseDto {
    /** Код игры */
    GameId: string;
    /** Статус игры */
    Status: number;
    /** Наименование комнаты */
    Name: string;
}

File: HorseRacing.Contracts/Models/Game/Responses/CreateGame/CreateGameResponseDto.cs
﻿using HorseRacing.Contracts.Base.Dto;
using HorseRacing.Domain.GameAggregate.Enums;

namespace HorseRacing.Contracts.Models.Game.Responses.CreateGame
{
    /// <summary>
    /// Модель игры
    /// </summary>
    public class CreateGameResponseDto : BaseDto
    {
        /// <summary>
        /// Код игры
        /// </summary>
        public Guid GameId { get; set; }
        /// <summary>
        /// Статус игры
        /// </summary>
        public StatusType Status { get; set; }
        /// <summary>
        /// Наименование комнаты
        /// </summary>
        public string Name { get; set; } = string.Empty;
    }
}
